fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1629,11 +1629,15 @@ relationPrimary
     | '(' VALUES rowConstructor (',' rowConstructor)* ')'
         (AS? alias=identifier columnAliases?)?                                          #inlineTable
     | subquery (AS? alias=identifier columnAliases?)?                                   #subqueryWithAlias
-    | qualifiedName '(' expressionList ')'
-        (AS? alias=identifier columnAliases?)?                                          #tableFunction
+    | tableFunctionCall (AS? alias=identifier columnAliases?)?                          #tableFunction
+    | TABLE '(' tableFunctionCall ')' (AS? alias=identifier columnAliases?)?            #tableFunctionTable
```

Comment:
Can you unify tableFunctionTable and NormalizedTableFunctionRelation into one name?